docs/_quarto.yml

```diff
@@ -138,9 +130,16 @@ website:
       style: "docked"
       collapse-level: 2
       contents:
+        - section: Backend API
+          contents:
+            - support_matrix.qmd
+            - reference/connection.qmd
+            - section: Specific Backend APIs
+              contents:
+                - auto: backends/*.qmd
```

Comment:
I like our existing backend layout.

Can remove this section and the separate connection section?

How about a how-to that describes how to use `connect`, `set_backend` and `get_backend` instead, that shows their API.

Alternatively, we can have a page that has the base backend API, along with these top level functions.

Either of these can be done in a follow-up.